source/access/text_mode_access_using_putty.rst

```diff
@@ -28,42 +37,80 @@ VSC clusters.
       <hardware>`, and replace ``login.hpc.kuleuven.be`` accordingly.
 
 
-#. Within the category Session, in the field 'Host Name', type in
-   <vsc-loginnode>, which is the name of the login node of the VSC
-   cluster you want to connect to.
+- Within the category Session, in the field 'Host Name', type in
+  <vsc-loginnode>, which is the name of the login node of the VSC
+  cluster you want to connect to.
 
```

Comment:
You can set an additional link to the hardware page mentioning that they can find the name of the login node on that page.